src/index.html

```diff
@@ -11,7 +11,208 @@
+          value="12"
+        >
+      </label>
+      <label
+        for="date"
+        class="label"
+      >
+      Full date of birth:
+        <input
+          type="date"
+          id="date"
+          required
+          class="input"
+        >
+      </label>
+      <label
+        for="terms"
+        class="label no_margin"
+      >
+      I accept the term of the agreement
+        <input
+          type="checkbox"
+          id="terms"
+          required
+        >
+      </label>
+    </fieldset>
+    <fieldset class="fieldset">
+      <legend>Registration</legend>
+      <label
+        for="email"
+        class="label"
+      >
+      E-mail:
+        <input
+          type="email"
+          id="email"
+          required
+          placeholder="email@example.com"
+          class="input"
+        >
+      </label>
+      <label
+        for="password"
+        class="label no_margin"
+      >
+      Password:
+        <input
+          type="password"
+          required
+          id="password"
+          minlength="5"
+          maxlength="15"
+        >
+      </label>
+    </fieldset>
+    <fieldset class="fieldset">
+      <legend>An interesting fact about you!</legend>
+      <label
+        for="cat"
+        class="label"
+      >
+      Do you love cats?
+        <input
+          type="radio"
+          id="cat"
+          name="question"
+          value="Yes"
+          class="input"
+        >
+        Yes
+        <input
+          type="radio"
+          name="question"
+          value="No"
+          class="input"
+        >
+        No
+      </label>
+      <label
+        for="color"
+        class="label"
+        >
+        What is your favorite color?
+        <input
+          type="color"
+          id="color"
+          class="input"
+        >
+      </label>
+      <label
+        for="sleep"
+        class="label"
+      >
+        What time do you go to bed?
+        <input
+          type="time"
+          id="sleep"
+          class="input"
+        >
+      </label>
+      <label
+        for="cars"
+        class="label"
+      >
+        What are your favorite brand of cars?
+        <select
+          name="cars"
+          id="cars"
+          multiple
+        >
+          <option value="BMW">BMW</option>
+          <option value="Audi">Audi</option>
+          <option value="Lada">Lada</option>
+        </select>
+      </label>
+      <label
+        for="range"
+        class="label no_margin"
+      >
+        How do you rate our work?
+        <input
+          type="range"
+          class="range"
+          value="0"
+        >
+      </label>
+    </fieldset>
+    <fieldset class="fieldset">
+      <legend>Additional info:</legend>
+      Comments:
+      <textarea
+      name="textarea"
+      id="textarea"
+      cols="20"
+      rows="2"
+      class="textarea"
+     ></textarea>
+      <label
+        for="recomendation"
+        class="label no_margin">
```

Comment:
```suggestion
      <label
        for="recomendation"
        class="label no_margin"
      >
```

A closing bracket should be at the new line if you have multiline attributes.